src/components/lib/ConnectWalletDialog.js

```diff
@@ -15,6 +16,20 @@ const ledgerConnector = new LedgerConnector({
 	url: ETH_RPC_URL
 })
 
+const trezorConnector = new TrezorConnector({ 
+	chainId: 1,
```

Comment:
I think here, we should also pass `CHAIN_ID`. We address a larger network id's issue with a custom implementation of `signTransactionAsync`.